README.md

```diff
@@ -29,9 +29,14 @@ npm install && cd server && npm install && cd ..
 ```
 ## Adding configuration
 
-Since the current widgets fetch data remotely, **some configuration parameters are required**. Configuration is loaded from `src/config/config.[env].js`, where `env` can be either `dev` or `prod`. 
+Since the current widgets fetch data remotely, **some configuration parameters are required**. According to [best practices](https://devcenter.heroku.com/articles/node-best-practices#be-environmentally-aware) the configuration is loaded from a `server/.env` in the project's folder for local development. In production the configuration should be set as environment variables.
 
-- Copy `config.default.js` to a new file named `config.dev.js`, open it, and fill-out the required API keys, etc. accordingly. 
+- Create a file named `server/.env`, open it, and fill-out the required API keys, etc. accordingly, e.g.
```

Comment:
Please write `credentials` instead of `API keys, etc. accordingly`, it's shorter and more specific.